lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -152,13 +446,35 @@ function writeStringsToLocaleFormat(locale, strings) {
   fs.writeFileSync(fullPath, JSON.stringify(output, null, 2) + '\n');
 }
 
-const strings = collectAllStringsInDir(path.join(LH_ROOT, 'lighthouse-core'));
-const psuedoLocalizedStrings = createPsuedoLocaleStrings(strings);
-console.log('Collected from LH core!');
+// @ts-ignore Test if called from the CLI or as a module.
+if (require.main === module) {
+  const strings = collectAllStringsInDir(path.join(LH_ROOT, 'lighthouse-core'));
+  console.log('Collected from LH core!');
+
+  collectAllStringsInDir(path.join(LH_ROOT, 'stack-packs/packs'), strings);
+  console.log('Collected from Stack Packs!');
 
-collectAllStringsInDir(path.join(LH_ROOT, 'stack-packs/packs'), strings);
-console.log('Collected from Stack Packs!');
+  if ((collisions) > 0) {
+    // TODO(exterkamp): assert that this number is constant to prevent unexpected dupes.
```

Comment:
what's the plan here? Can we just assert a constant right here until you come up with something else?

Reply:
Yeah plan was to assert/test in follow up. But easy enough to do here, good catch!